File: C#/exercicio03/solucao.cs
using System;

public class Program
{
    public static void Main()
    {
        OlaFuncionario();
    }

    static void OlaFuncionario()
    {
        Console.WriteLine("Digite o seu nome: ");
        string nome = Console.ReadLine(); 

        Console.WriteLine("Digite o valor do seu salario:" );
        string input = Console.ReadLine();
        int salario = int.Parse(input);

        Console.WriteLine($"O funcionario {nome}, tem um salario de {salario}");

        Console.WriteLine("Pressione Enter para sair...");
        Console.ReadLine();
    }
}

File: Python/exercicio-13/solucao.py
def main():
    def aumento_salario():
        salario_atual = float(input("Digite o valor do seu salario:  " ))
        aumento = salario_atual * 0.15
        salario_com_aumento = salario_atual + aumento

        print(f'O seu novo salario com aumento de 15% é: {salario_com_aumento} Reais')
    

    aumento_salario()

main()
input('Pressione Enter para sair ...')